src/xcode/ENA/ENA/Source/Scenes/ENSetting/EUSettings/Cells/FlagIconCell.swift

```diff
@@ -0,0 +1,50 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import Foundation
+import UIKit
+
+/// - NOTE: The implementation may raise a 'kCFRunLoopCommonModes' warning that is a known UIKit bug: https://developer.apple.com/forums/thread/132035
+class FlagIconCell: UITableViewCell {
```

Comment:
I have basically the same requirement in my story for the submission flow and just used the IconCell, maybe we could also use it here? Or maybe you can convince me to use this one as well 😜

Reply:
Ah, perfect! In that case I will change it to the IconCell. :^)